app/views/shared/partials/_media_contents_form.html.erb

```diff
@@ -48,7 +48,7 @@
               </div>
             <% end %>
 
-            <div class="row">
+            <div class="row" hidden>
```

Comment:
I would have converted it to a hidden input, but since it's in the CMS, I'm not picky here

Reply:
good alternative, yes